environs/instances/instancetype.go

```diff
@@ -210,3 +211,29 @@ func (s byMemory) Less(i, j int) bool {
 	// Result is in descending order of cost so instance with lowest cost is used.
 	return inst0.Cost > inst1.Cost
 }
+
+//by Type is used to sort a slice by name by best effort. As we have different separators for different providers
```

Comment:
"ByName" is used...

Nit: can we get a space between the slashes and the text?

Reply:
Oh thanks, I missed that!